crates/services/sync/src/import/tests.rs

```diff
@@ -592,6 +592,9 @@ impl DefaultMocks for MockBlockImporterPort {
 pub(crate) fn empty_header(h: BlockHeight) -> SourcePeer<SealedBlockHeader> {
     let mut header = BlockHeader::default();
     header.consensus.height = h;
+    let mut transaction_tree =
+        fuel_core_types::fuel_merkle::binary::in_memory::MerkleTree::new();
+    header.application.generated.transactions_root = transaction_tree.root().into();
```

Comment:
cc @bvrooman for some reason the root returned here doesn't match the root in the data-driven test suite

Reply:
What we actually get: `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855`

What is in the test specs for bmt with zero leaves: `ce04537ec4b4b84cf02fa5c50c7810e2f6592e3b00de7423d7c3e7e7d8689a67`

Reply:
Discussed on Slack: `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` is actually the correct root in this case - it is the sha256 hash of the empty string.